pkg/operation/botanist/component/vpnseedserver/vpn_seed_server.go

```diff
@@ -567,6 +582,183 @@ func (v *vpnSeedServer) Deploy(ctx context.Context) error {
+		}
+		envoyFilter.Spec.ConfigPatches = []*istionetworkingv1alpha3.EnvoyFilter_EnvoyConfigObjectPatch{
+			&istionetworkingv1alpha3.EnvoyFilter_EnvoyConfigObjectPatch{
+				ApplyTo: istionetworkingv1alpha3.EnvoyFilter_NETWORK_FILTER,
+				Match: &istionetworkingv1alpha3.EnvoyFilter_EnvoyConfigObjectMatch{
+					Context: istionetworkingv1alpha3.EnvoyFilter_GATEWAY,
+					ObjectTypes: &istionetworkingv1alpha3.EnvoyFilter_EnvoyConfigObjectMatch_Listener{
+						Listener: &istionetworkingv1alpha3.EnvoyFilter_ListenerMatch{
+							Name:       "0.0.0.0_8132",
+							PortNumber: 8132,
+							FilterChain: &istionetworkingv1alpha3.EnvoyFilter_ListenerMatch_FilterChainMatch{
+								Filter: &istionetworkingv1alpha3.EnvoyFilter_ListenerMatch_FilterMatch{
+									Name: "envoy.filters.network.http_connection_manager",
+								},
+							},
+						},
+					},
+				},
+				Patch: &istionetworkingv1alpha3.EnvoyFilter_Patch{
+					Operation: istionetworkingv1alpha3.EnvoyFilter_Patch_MERGE,
+					Value: &protobuftypes.Struct{
+						Fields: map[string]*protobuftypes.Value{
+							"name": &protobuftypes.Value{
+								Kind: &protobuftypes.Value_StringValue{
+									StringValue: "envoy.filters.network.http_connection_manager",
+								},
+							},
+							"typed_config": &protobuftypes.Value{
+								Kind: &protobuftypes.Value_StructValue{
+									StructValue: &protobuftypes.Struct{
+										Fields: map[string]*protobuftypes.Value{
+											"@type": &protobuftypes.Value{
+												Kind: &protobuftypes.Value_StringValue{
+													StringValue: "type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager",
+												},
+											},
+											"route_config": &protobuftypes.Value{
+												Kind: &protobuftypes.Value_StructValue{
+													StructValue: &protobuftypes.Struct{
+														Fields: map[string]*protobuftypes.Value{
+															"virtual_hosts": &protobuftypes.Value{
+																Kind: &protobuftypes.Value_ListValue{
+																	ListValue: &protobuftypes.ListValue{
+																		Values: []*protobuftypes.Value{
+																			{
+																				Kind: &protobuftypes.Value_StructValue{
+																					StructValue: &protobuftypes.Struct{
+																						Fields: map[string]*protobuftypes.Value{
+																							"name": &protobuftypes.Value{
+																								Kind: &protobuftypes.Value_StringValue{
+																									StringValue: v.namespace,
+																								},
+																							},
+																							"domains": &protobuftypes.Value{
+																								Kind: &protobuftypes.Value_ListValue{
+																									ListValue: &protobuftypes.ListValue{
+																										Values: []*protobuftypes.Value{
+																											{
+																												Kind: &protobuftypes.Value_StringValue{
+																													StringValue: *v.kubeAPIServerHost + ":8132",
```

Comment:
Where is `8132` coming from? I see it multiple times, maybe we should have a constant somewhere?

Reply:
Introduced a constant.